packages/shared/providers.yaml

```diff
@@ -1485,6 +1485,21 @@ marketo:
     token_params:
         grant_type: client_credentials
     docs: https://docs.nango.dev/integrations/all/marketo
+medallia:
+    categories:
+        - crm
+        - support
+        - surveys
+    auth_mode: OAUTH2_CC
+    token_url: https://${connectionConfig.REPORTING_INSTANCE}/oauth/${connectionConfig.TENANT_NAME}/token
+    token_request_auth_method: basic
+    token_params:
+        grant_type: client_credentials
+    proxy:
+        retry:
+            after: 'X-RateLimit-Reset'
+        base_url: https://${connectionConfig.GATEWAY_URL}.apis.medallia.com
```

Comment:
Just a question, sometimes we use camelcase sometimes uppercase. Is there any guidelines?

Reply:
@bodinsamuel , just updated the PR with `camelCase` as it is what is mainly used. Thank you for catching that.

Reply:
thanks, it was more a general question than a suggestion :D